docs/cli/kops_completion.md

```diff
@@ -0,0 +1,38 @@
+## kops completion
+
+Outputs kops shell completion for the given shell (bash)
+
+### Synopsis
+
+
+Outputs kops shell completion for the given shell (bash)
+
+```
+kops completion
+```
+
+### Options
+
+```
+      --shell string   target shell (bash).
```

Comment:
There is an `Example` field you can use - https://godoc.org/github.com/spf13/cobra#Command

Will add an example for the `completion` command, and yeah, let's create a new issue to add `Example` to the current commands.